platform/admin/ui/src/main/webapp/js/views/configuration/ConfigurationEdit.view.js

```diff
@@ -255,9 +257,29 @@ define([
         /**
          * Submit to the backend.
          */
-        submitData: function() {
+        submitData: function () {
+            var spinner = new Spinner(define({
```

Comment:
It's tweaked a bit from the defaults in spinnerConfig.js, mainly changed the color to contrast better with the modal background. I could still use a modified spinnerConfig.js here, if that wouldn't be too confusing. Thoughts?